cypress/integration/app/queryflow.spec.ts

```diff
@@ -0,0 +1,57 @@
+describe('Query flow', () => {
+  beforeEach(() => {
+    cy.setup();
+    cy.login();
+  });
+
+  it('Create, check, edit, and delete a query successfully', () => {
+    cy.visit('/queries/manage');
+
+    cy.contains('button', /create new query/i).click();
+
+    cy.get('.query-form__query-title').type('Query all window crashes');
```

Comment:
Rather than using CSS selectors, let's prefer to target elements based on what the user actually sees:

```suggestion
    cy.findByText(/title/i).next('input').type('Query all window crashes')
```

This is motivated primarily by the fact that we don't want the tests coupled to the selectors themselves, but rather the actual content that the user would interact with. This way, even if we were to change the styling of these elements, the test would continue to succeed.

Reply:
In addition to the selectors available directly in Cypress, we are also including the selectors from https://github.com/testing-library/cypress-testing-library#usage which can help with this style of testing.

Note I tried using the `findByLabelText` and that didn't work though it seems like it ought to in this case. Instead using `findByText` combined with `next` seemed to work well.

Reply:
Ok, I refactored almost all of the .get to user-facing grabs and left comments for anything that was still wonky. All 3 tests go through the entire flow. @ghernandez345 @zwass